src/App.tsx

```diff
@@ -1,63 +1,116 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import classNames from 'classnames';
+import { GoodsList } from './components/GoodsList';
 
-export const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
+import goodsFromServer from './data/goods.json';
+
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+export function getReorderedGoods(
+  goods: string[],
+  isReversed: boolean,
+  sortType: SortType,
+) {
```

Comment:
When working with functions that have a lot of arguments, it can sometimes be difficult to remember the order of the arguments or even to know what each argument is for. To improve the readability and maintainability of your code, it's recommended to group related arguments together in an options object instead.

In the example provided, `isReversed` and `sortType` are two arguments that are related to the sorting of goods. Instead of passing them as separate arguments, we can group them together in an options object:

```js
export interface GoodsSortingOptions {
  isReversed: boolean;
  sortType: SortType;
}

export function getReorderedGoods(
  goods: string[],
  options: GoodsSortingOptions
) {
...
}
```

This approach has several advantages:

- **Readability**: It's easier to understand what each argument is for when they are grouped together in an options object.

- **Flexibility**: We can add new options without breaking existing code. For example, if we want to add a `maxItems` option, we can simply add it to the `GoodsSortingOptions` interface.

- **Reduced cognitive load**: When calling the function, we only need to pass in a single argument instead of multiple arguments.

Here's an example of how we can call the function with the new options object:

```js
const goods = ["apple", "banana", "orange"];
const options = {
  isReversed: true,
  sortType: SortType.ASCENDING,
};

const reorderedGoods = getReorderedGoods(goods, options);
```

In summary, grouping related arguments together in an options object can improve the readability and maintainability of your code, and reduce the cognitive load of calling the function.